pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -966,6 +966,42 @@ public void getProperties(
                 });
     }
 
+    @PUT
+    @Path("/{tenant}/{namespace}/{topic}/properties")
+    @ApiOperation(value = "Replace all the properties on the given subscription")
```

Comment:
`Replace all the properties`  -> `Update the properties`

Reply:
Nice catch!  Updated